lib/Runtime/Debug/TTEventLog.cpp

```diff
@@ -1199,7 +1199,7 @@ namespace TTD
 
     bool EventLog::IsTimeForSnapshot() const
     {
-        return (this->m_elapsedExecutionTimeSinceSnapshot > this->m_threadContext->TTDContext->SnapInterval);
+        return (this->m_elapsedExecutionTimeSinceSnapshot >= this->m_threadContext->TTDContext->SnapInterval);
```

Comment:
What's the reason for this? Why would we want to take a snapshot after no time has elapsed? Or is the elapsed time limited in precision and so in some tests we take snapshots too quickly and they can't be distinguished?

Reply:
This was the fix for one of the osx/linux failures.  In this case, if `SnapInterval` is 0, it means we want to take a snapshot at every opportunity, without respect to how much time has elapsed since the last snapshot.  On Linux & OSX, `m_elapsedExecutionTimeSinceSnapShot` was 0, meaning we'd skip these snapshots.  This meant that we had a different number of snapshots on windows & Linux, causing differences in test behavior. 

Maybe more clear if this code was (`return this->m_threadContext->TTDContext->SnapInterval == 0 || (this->m_elapsedExecutionTimeSinceSnapshot > this->m_threadContext->TTDContext->SnapInterval);`

Reply:
Ah yes, that does make it more clear in my opinion.

Reply:
OK, updated (commit is squashed).